shaded-ning19/pom.xml

```diff
@@ -23,13 +23,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.apex</groupId>
-    <artifactId>apex</artifactId>
-    <version>3.3.0-incubating</version>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+    <relativePath/>
   </parent>
   <groupId>org.apache.apex</groupId>
   <artifactId>apex-shaded-ning19</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
Separating into a separate repo would incur the cost of another apache
build iteration for these but at least that will be at a much smaller
frequency. Long term this maybe the better option.

On Sep 2, 2016, at 12:33 PM, Vlad Rozov <notifications@github.com> wrote:

In shaded-ning19/pom.xml
<https://github.com/apache/apex-core/pull/380#discussion_r77400947>:

>    </parent>
>    <groupId>org.apache.apex</groupId>
>    <artifactId>apex-shaded-ning19</artifactId>
> -  <version>1.0.0</version>
> +  <version>1.1.0-SNAPSHOT</version>

The same question as for codestyle-config. Do we want to release Apex Core
with SNAPSHOT version? If not, will it be necessary to release
shaded-ning19 before the next Apex Core release?

codestyle-config and shaded ning may technically reside in a different git
repository and not be part of the Apex core source release. IMO, it will be
better to avoid any dependency of codestyle-config and shaded ning on the
Apex core including commits.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/apache/apex-core/pull/380/files/20758fa06ff6e1a1f4ba224d39025e4dc8999d0e#r77400947>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AENqu5MWe4bPhTZjI6RBQdWBWCBuihHlks5qmHokgaJpZM4Jz-Ur>
.